addons/website_sale_comparison/static/src/scss/website_sale_comparison.scss

```diff
@@ -31,3 +94,25 @@
         @include font-size(1.1rem);
     }
 }
+
+.o_wsale_comparison_bottom_bar {
+    .o_product_image {
+        width: 2.5rem;
+    }
+
+    .o_wsale_comparison_offcanvas_image {
+        width: 5rem;
+    }
+
+}
+
+// Style the <main> element when comparison bar is present
+:where(main:has(.o_wsale_comparison_bottom_bar)) {
+    position: relative;
+    padding-bottom: 80px;
```

Comment:
Woops missed this one, I think it would be better to use a css var so you can use it in your calc and it's clear that these values are linked